tests/e2e/tutorials/ti-plan.js

```diff
@@ -65,11 +66,59 @@ async function runTutorial() {
       await utils.waitAndClick(electrodeSelectorIframe, `[osparc-test-id="Electrode_${grp[1]}"]`);
     }
     await utils.waitAndClick(electrodeSelectorIframe, `[osparc-test-id="FinishSetUp"]`);
+    await tutorial.waitFor(5000, "Finish Electrode Selector SetUp");
+    await tutorial.takeScreenshot("electrodeSelector_after");
 
     // Run optimizer
     await tutorial.waitAndClick("AppMode_NextBtn");
     await tutorial.waitFor(5000, "Running Optimizer");
-    await tutorial.waitForStudyDone(studyId, 120000);
+    await tutorial.takeScreenshot("optimizer_before");
+    // one permutation should take less than 180"
+    await tutorial.waitForStudyDone(studyId, 240000);
+    await tutorial.takeScreenshot("optimizer_after");
+    await tutorial.waitAndClick("preparingInputsCloseBtn");
+    await tutorial.waitFor(5000, "Optimizer Finished");
+
+    // Load Post Pro Analysis
+    await tutorial.takeScreenshot("postpro_start");
+    const postProIframe = await tutorial.getIframe(workbenchData["nodeIds"][2]);
+    // Click "Load Analysis" button
+    const buttonsLoadAnalysis = await utils.getButtonsWithText(postProIframe, "Load Analysis");
+    await buttonsLoadAnalysis[0].click();
+    await tutorial.waitFor(10000, "Loading anaylsis");
+    await tutorial.takeScreenshot("postpro_load_analysis");
+    // Click on the first "Load" button
```

Comment:
why not to log these step messages?

Reply:
The second argument in the ```waitFor``` gets logged 👍